blueprints/codes/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""Module for the abstract base class Equation."""
+from abc import ABC, abstractmethod
+
+
+class Equation(ABC):
```

Comment:
Why dit we choose "Equation"? it doesn't seem corresponding to the definitions in math: https://www.mathswithgraham.org.uk/?p=2873

I'm happy to choose whatever name we want: but I think it is a good idea to document the design choice in the Issue #4  so we can look it up later and avoid later discussions.

Reply:
@bro-wi sharp!! In this case it should be Formula instead of Equation. I will change it 👌